index.html

```diff
@@ -7,7 +7,6 @@
     <link rel="stylesheet" href="css/style.css" />
 </head>
 <body>
-    <div class="fcc_test_ui"><div id="fcc_foldout_toggler"> <span id="hamburger_top" class="fcc_hamburger"></span> <span id="hamburger_middle" class="fcc_hamburger"></span> <span id="hamburger_bottom" class="fcc_hamburger"></span> <span id="fcc_foldout_toggler_background"></span> </div> <input id="fcc_toggle" onclick="FCC_Global.hamburgerTransform()" type="checkbox" title="CTRL + SHIFT + O"> <div id="fcc_test_suite_indicator_wrapper"><span id="fcc_test_suite_indicator">FCC Test Suite: Product Landing Page</span></div> <div id="fcc_foldout_menu"><div id="fcc_foldout_menu_inner"> <label for="test-suite-selector">Select Test Suite:</label> <select name="Test Suite Selector" id="test-suite-selector" onchange="FCC_Global.selectProject(this.value)" disabled=""> <option id="placeholder" value="">- - -</option> <option value="tribute-page">Tribute Page</option><option value="portfolio">Personal Portfolio</option><option value="survey-form">Survey Form</option><option value="product-landing-page">Product Landing Page</option><option value="technical-docs-page">Technical Documentation Page</option><option value="random-quote-machine">Random Quote Machine</option><option value="markdown-previewer">Markdown Previewer</option><option value="drum-machine">Drum Machine</option><option value="25-5-clock">25 + 5 Clock</option><option value="javascript-calculator">Javascript Calculator</option><option value="bar-chart">D3: Bar Chart</option><option value="scatter-plot">D3: Scatter Plot</option><option value="heat-map">D3: Heat Map</option><option value="choropleth">D3: Choropleth</option><option value="tree-map">D3: Tree Map</option></select> <div id="fcc_test_buttons_wrapper"> <button id="fcc_test_message-box-rerun-button" type="button" class="fcc_foldout_button" title="CTRL + SHIFT + ENTER" onclick="FCC_Global.FCCRerunTests()"> Run Tests </button> <button id="fcc_test_button" type="button" class="fcc_foldout_button fcc_test_btn-default fcc_test_btn-done fcc_test_btn-success" title="CTRL + SHIFT + T" onclick="FCC_Global.FCCOpenTestModal()">16/16 Passed</button> </div> <div id="fcc_legend_wrapper"> <div class="fcc_legend key"></div> <span class="fcc_legend">Test(s) Failed</span> <div class="fcc_legend key"></div> <span class="fcc_legend">Tests Passed</span> <div class="fcc_legend key"></div> <span class="fcc_legend">Tests Executing</span> </div> <span id="fcc_ask-for-help"><a href="https://forum.freecodecamp.org/c/general" target="_blank">Ask For Help</a> </span> </div> </div></div>
     <div id="page-wrapper">
```

Comment:
Please consider separating the header from the wrapper div. Separating it will help differentiate the header from the main content.